### PR TITLE
Convert the datapusher to work as plugin

### PR DESCRIPTION
**Pull request by tino097**
Follow ckan extension fodler structure
Add plugins.py 
Move the logic to coresponding folders/script

### CODE REVIEW

**Comment by tino097** [comment]
@TomeCirun you can pickup from this branch and push your changes directly to this one

File: ckanext/datapusher_plus/jobs.py
Comment (jqnatividad) [inline ckanext/datapusher_plus/jobs.py:1]: @tino097 , like how the core of jobs.py was kept.

File: setup.py [hunk 1]
```diff
@@ -51,20 +53,16 @@
...
-        'ckanserviceprovider == 1.1.0',
-        'requests',
-        "psycopg2",
-        'datasize',
-        'semver',
-        'uwsgi',
+        # CKAN extensions should not list dependencies here, but in a separate
+        # ``requirements.txt`` file.
+        #
+        # http://docs.ckan.org/en/latest/extensions/best-practices.html#add-third-party-libraries-to-requirements-txt
```
Comment (jqnatividad) [inline setup.py:65]: Thanks for cleaning this up.

**Review by jqnatividad — APPROVED** [review]
LGTM @tino097 !

**Pull request merged**